tests/integration/cmd_namespace_test.go

```diff
@@ -178,7 +178,7 @@ ComponentSettings:
 			})
 		})
 
-		Describe("list "+commandName, func() {
+		Describe("list "+commandName, FlakeAttempts(3), func() {
```

Comment:
We should avoid using this, as it will result in more flaky tests, and so much longer execution times
I proposed another solution in https://github.com/redhat-developer/odo/pull/5837

Reply:
Understood! Will revert these changes.